lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,28 @@
+module Semian
+  class AtomicInteger #:nodoc:
+    attr_accessor :value
+
+    def initialize(_name, _permissions)
```

Comment:
Perhaps with keyword arguments? For example:
```ruby
class AtomicEnum
  def initialize(symbol_list)
  ...

class SysVAtomicEnum < AtomicEnum
  def initialize(symbol_list, name: , permissions: )
  ...

```
This would work. Would there be any compatibility issues?